common/src/main/java/com/netflix/conductor/common/utils/EnvUtils.java

```diff
@@ -14,12 +14,24 @@
 
 import java.util.Optional;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class EnvUtils {
 
+    private static final Logger logger = LoggerFactory.getLogger(EnvUtils.class);
+
     public static final String ELASTIC_SEARCH_HEALTH_URL = "elasticsearch.health.url";
     public static final String ELASTIC_SEARCH_DEFAULT_HEALTH_URL = "/_cluster/health";
     public static final String INDEX_ON_COMMUNICATION_FAILURE = "index.on.communication.failure";
 
+    static {
+        logger.info(
+                "{} value set to {}",
+                EnvUtils.INDEX_ON_COMMUNICATION_FAILURE,
+                indexOnCommunicationFailure());
```

Comment:
Instead of logging this in static block, is it possible to log it inside a init method (post construct) somewhere ?